sqlmesh/core/table_diff.py

```diff
@@ -52,6 +52,8 @@ class RowDiff(PydanticModel, frozen=True):
     target: str
     stats: t.Dict[str, float]
     sample: pd.DataFrame
+    source_env: str = "source"
```

Comment:
Do we even need default values for source and env names? Should they just be optional?

Reply:
same, resolved